Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -1,127 +1,126 @@
 
-# Stop the stack
-bash ./stack.sh stop <STACK_NAME>
+```
 
-# Remove stack services (incl. volumes)
-bash ./stack.sh remove <STACK_NAME> -v
+./stack.sh start <STACK NAME> <PORT>
 
+```
 
-After spinning up the stack, the GUI endpoints to the running containers can be accessed via Browser (i.e. adminer, blazegraph, ontop, geoserver). The endpoints and required log-in settings can be found in the [spin up the stack] readme.
+# 4. Using the Agent
 
-&nbsp;
-## 1.3 Deploying the agent to the stack
+The agent will automatically register a task upon startup to assimilate the data from the target GPS folder into the Knowledge Graph (KG). This background task can be triggered via an HTTP request after the agent has started, accessible at http://localhost:{Port}/fenland-trajectory-agent. Please replace {Port} with the numerical value of the port on which your stack is running.
 
-As mentioned earlier, this agent requires [JPS_BASE_LIB] and [Stack-Clients] to be wrapped by [py4jps]. Therefore, after installation of all required packages (incl. `py4jps >= 1.0.32`), the `StackClients` resource needs to be added to allow for access through `py4jps`. All required steps are detailed in the [py4jps] documentation. However, the commands provided below shall suffice to compile the latest `StackClients` resource locally and install it inside the Docker container using the provided [Dockerfile]. If you encounter any error for spinning up the stack, please manually execute commands mentioned in Stack Client Settings section in this page.
+## Provided functionality
 
-Simply execute the following command in the same folder as this `README` to build the required [Stack-Clients] resource and spin up the production version of the agent (from a bash terminal). The stack `<STACK NAME>` is the name of an already running stack.
+The files for example http request are displayed at [SendHTTP] folder.
 
+<center>-------- Data Instantiation ------------</center>
 
-# Buildings the agent Docker image and pushing it
-bash ./stack.sh build
+- Post request to load and preprocess all GPS trajectory files 
+> ./gpstasks/fenlandtrajectoryagent/load_and_preprocess
 
-# Deploying the agent (using pulled image)
-bash ./stack.sh start <STACK_NAME>
+- Post request to instantiate all GPS trajectory data to KG
+> ./gpstasks/fenlandtrajectoryagent/process_and_instantiate
 
+Additionally, services above can be triggered using [CURL commands] from a bash terminal. This method provides a straightforward and scriptable way to interact with the agent. An example CURL command used to load the GPS trajectory files is displayed as follows. In this example, the stack is assumed at port 3840
 
-&nbsp;
-# 2. Using the Agent
- 
-## Provided functionality
+```
+curl -X POST http://localhost:3840/fenland-trajectory-agent/gpstasks/fenlandtrajectoryagent/load_and_preprocess \
+     -H "Content-Type: application/json" \
+     -d '{"file_path":"/app/agent/raw_data/gps_target_folder"}'
```

Comment:
Rather than repeating the content of your `sh` and/or `http` files in this readme (and thus running the risk that these will get out of sync), I would recommend simply pointing to the location of these files here.